.github/scripts/metrics-replace-annotation.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
instead of adding a new script in  .github I will add them in the respective operators MAKEFILEs

Reply:
done